<html>
	<head>
		<title>Laboratory of Computational Proteomics - Research: Identification</title>
		<link rel="stylesheet" type="text/css" href="/css/Style.css" />
	</head>

	<body bgcolor="#ffffff" bottommargin="0" leftmargin="0" marginheight="0" marginwidth="0" rightmargin="0" topmargin="0">
		<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
			<tr>
				<td>
					<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
						<tr>
							<td width="2"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="2" height="5" border="0"></td>
							<td width="6"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="6" height="5" border="0"></td>
							<td width="172" height="50" bgcolor="#EEEEEE" align="left" valign="middle"><img src="/images/logo.png" width="172" height="43" border="0"></td>
							<td width="1"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="4" height="5" border="0"></td>
							<td width="620" class="siteName" align="center">Laboratory of Computational Proteomics<br><font size=-1>Research: Identification</font></td>
							<td width="5"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
						</tr>
					</table>
				</td>
			</tr>
			<tr>		
				<td valign="top">
					<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
						<tr>
							<td width="10"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="10" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="160" height="5" bgcolor="#AAAAAA" align="center" valign="top"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="620" height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="376" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td  height="5" bgcolor="#EEEEEE" valign="top"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
						</tr>
						<tr>
							<td width="10" bgcolor="#EEEEEE"><img src="/images/spacer.gif" border="0"></td>
							<td width="5"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td width="160" valign="top" bgcolor="#EEEEEE" align="center">
								<table width="150" border="0" cellpadding="0" cellspacing="0" bgcolor="#EEEEEE">
									<tr>
										<td style="padding: 20 0 20 8;" align="center">
											<table width="100%" border="0" cellpadding="0" cellspacing="8">
												<tr>
													
													<td><a class="menu" href="/index.html">Overview</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/research/research.html">Research</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/members/members.html">Members</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/publications/publications.html">Publications</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/presentations/presentations.html">Presentations</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/tools/tools.html">Tools</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/contact/contact.html">Contact</a></td>
												</tr>
												<tr>
													<td>&nbsp;</td>
												</tr>
											</table>
										</td>
									</tr>
									<tr>
										<td style="padding: 0 0 0 0;" align="left">
											<table width="120" border="0" cellpadding="0" cellspacing="0">	
												<tr>
													<td><center>
														<a class="chibi" href="http://www.nyuinformatics.org/" target="_blank">Center for Health Informatics and Bioinformatics</font></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.med.nyu.edu/" target="_blank"><img src="/images/sponsors/logo_nyulmc_blue.jpg" alt="NYU Langone Medical Center" width="97" height="51" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nih.gov/" target="_blank"><img src="/images/sponsors/nih-logo.gif" alt="NIH" width="63" height="63" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://cancer.gov/" target="_blank"><img src="/images/sponsors/nci-logo.jpg" alt="NCI" width="89" height="56" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nigms.nih.gov/" target="_blank"><img src="/images/sponsors/nigms-logo.jpg" alt="NIGMS" width="63" height="75" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nidcr.nih.gov/" target="_blank"><img src="/images/sponsors/nidcr-logo.jpg" alt="NIDRC" width="58" height="72" vspace="2" border="0"></a>
													</td>
												</tr>
											</table>										
										</td>
									</tr>
								</table>
							</td>
							<td width="5"  height="5" bgcolor="#AAAAAA" ><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td width="620" style="padding: 25" valign="top" class="content">
							
								
								<b>Identification of proteins and characterization of their post-translational modifications.</b>
								<p>
								
								Mass spectrometry-based protein identification has become an
invaluable tool for elucidating protein function, and several methods
have been developed for protein identification, including
sequence collection searching with masses of peptides or their
fragments, spectral library searching, and de novo sequencing
(Fig. 1).
The first step in protein identification is to find peaks in the
mass spectra that correspond to peptides and their fragments. It
is important to find all the relevant peaks and at the same time
minimizing the number of background peaks. This can be
achieved by scanning the spectra for peaks of the expected width
and selecting peaks above a signal to noise threshold,
and then picking the monoisotopic peak for each isotope cluster. 
After picking the peaks, spectra with low information
content that could not produce any meaningful results can be
removed to increase the speed of subsequent analysis. 
<p>
In all mass spectrometry-based identification methods, a score
is calculated to quantify the match between the observed mass
spectrum and the collection of possible sequences. These scores
are highly dependent on the details of the algorithm used, and
they are not always easy to interpret because the interpretation of
the score depends on properties of the data and the search results.
Therefore, it is desirable to convert the score to a measure that is
easy to interpret, such as the probability that the result is random
and false. For this conversion, the distribution of random and false
scores is needed. Estimates of this distribution can be generated
using either simulations, collecting statistics during
the search, or direct calculations.
								<p>
								
								<table cellpadding="0" cellspacing="0" border="0"><tr>
								<td valign="top">
								<img alt="" src="identification-fig1.png" width="432" height="327">
								</td>
								</tr>
								<tr><td  valign="top"><b>Figure 1.</b> Mass spectrometry based workflows for protein identification : (a) searching a protein sequence collection with
								peptide mass information; (b) searching a protein sequence collection with peptide fragment mass information;
								(c) searching a spectrum library with peptide fragment mass information; (d) de novo sequencing.
								</td></tr>
								</table>
								
								<p>&nbsp;</p>
								
								
								<p id="reference"><strong>References</strong></p>
								<li>D. Feny&ouml;, J. Eriksson, R. Beavis, <a href="/publications/pdf/10_fenyo_methods-mol-biol.pdf" target="_blank"><strong>&quot;Mass spectrometric protein identification using the global proteome machine&quot;</a></strong>, Methods Mol Biol 673 (2010) 189-202.</li>
								<br>	
								<li>D. Feny&ouml;, B.S.Phinney, R.C. Beavis, <a href="/publications/pdf/07_fenyo_jpr.pdf" target="_blank"><strong>&quot;Determining the Overall Merit of Protein Identification Data Sets: rho-Diagrams and rho-Scores&quot;</a></strong>. <em>J Proteome Res</em> 6 (2007) 1997-2004.</li>
								<br>
								<li>R. Craig, J.C. Cortens, D. Feny&ouml;, R.C. Beavis, <a href="/publications/pdf/06_craig_jpr.pdf" target="_blank"><strong>&quot;Using Annotated Peptide Mass Spectrum Libraries for Protein Identification&quot;</a></strong> <em>Journal of Proteome Research</em>, 5 (2006) 1843-1849.</li>
								<br>
								<li>J. Eriksson, D. Feny&ouml; <a href="/publications/pdf/05_eriksson_jpr.pdf" target="_blank"><strong>&quot;Protein Identification in Complex Mixtures&quot;</a></strong>, <em>J Proteome Research</em> 4 (2005) 387-93.</li>
								<br>
								<li>D. Feny&ouml;, R.C. Beavis, <a href="/publications/pdf/03_fenyo_anal-chem.pdf" target="_blank"><strong>&quot;A method for assessing the statistical significance of mass spectrometry-based protein identifications using general scoring schemes&quot;</a></strong>, <i>Analytical Chemistry</i> 75 (2003) 768-74.</li>
								<br>
								<li>H.I. Field, D. Feny&ouml;, R.C. Beavis <a href="/publications/pdf/02_Field-Proteomics.pdf" target="_blank"><strong>&quot;RADARS, a bioinformatics solution that automates proteome mass spectral analysis, optimises protein identification, and archives data in a relational database&quot;</a></strong>, <i> Proteomics</i> 2 (2002) 36-47.</li>
								<br>
								<li>J. Eriksson, B.T. Chait, D. Feny&ouml; <a href="/publications/pdf/00_eriksson_anal-chem.pdf" target="_blank"><strong>&quot;A statistical basis for testing the significance of mass spectrometric protein identification results&quot;</a></strong> <i>Anal Chem</i> 72 (2000) 999-1005.</li>
								<br>
								<li>D. Feny&ouml;, J. Qin, B.T. Chait <a href="/publications/pdf/98_fenyo_electrophoresis.pdf" target="_blank"><strong>&quot;Protein Identification using Mass Spectrometric Information&quot;</a></strong> <i>Electrophoresis</i> 19 (1998) 998-1005.</li>
								<br>

								
								<p>&nbsp;</p>
								<center>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
							</td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td bgcolor="#EEEEEE"><img src="/images/spacer.gif" border="0"></td>
						</tr>
					</table>
				</td>
			</tr>
		</table>
	</body>

</html>
