<html>
	<head>
		<title>Laboratory of Computational Proteomics - Proteomics Informatics Spring 2013</title>
		<link rel="stylesheet" type="text/css" href="/css/Style.css" />
	</head>

	<body bgcolor="#ffffff" bottommargin="0" leftmargin="0" marginheight="0" marginwidth="0" rightmargin="0" topmargin="0">
		<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
			<tr>
				<td>
					<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
						<tr>
							<td width="2"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="2" height="5" border="0"></td>
							<td width="6"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="6" height="5" border="0"></td>
							<td width="172" height="50" bgcolor="#EEEEEE" align="left" valign="middle"><img src="/images/logo.png" width="172" height="43" border="0"></td>
							<td width="1"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="4" height="5" border="0"></td>
							<td width="620" class="siteName" align="center">Laboratory of Computational Proteomics<br><font size=-1>Proteomics Informatics Spring 2013</font></td>
							<td width="5"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
						</tr>
					</table>
				</td>
			</tr>
			<tr>		
				<td valign="top">
					<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
						<tr>
							<td width="10"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="10" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="160" height="5" bgcolor="#AAAAAA" align="center" valign="top"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="620" height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="376" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td  height="5" bgcolor="#EEEEEE" valign="top"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
						</tr>
						<tr>
							<td width="10" bgcolor="#EEEEEE"><img src="/images/spacer.gif" border="0"></td>
							<td width="5"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td width="160" valign="top" bgcolor="#EEEEEE" align="center">
								<table width="150" border="0" cellpadding="0" cellspacing="0" bgcolor="#EEEEEE">
									<tr>
										<td style="padding: 20 0 20 8;" align="center">
											<table width="100%" border="0" cellpadding="0" cellspacing="8">
												<tr>
													
													<td><a class="menu" href="/index.html">Overview</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/research/research.html">Research</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/members/members.html">Members</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/publications/publications.html">Publications</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/presentations/presentations.html">Presentations</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/tools/tools.html">Tools</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/contact/contact.html">Contact</a></td>
												</tr>
												<tr>
													<td>&nbsp;</td>
												</tr>
											</table>
										</td>
									</tr>
									<tr>
										<td style="padding: 0 0 0 0;" align="left">
											<table width="120" border="0" cellpadding="0" cellspacing="0">	
												<tr>
													<td><center>
														<a class="chibi" href="http://www.nyuinformatics.org/" target="_blank">Center for Health Informatics and Bioinformatics</font></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.med.nyu.edu/" target="_blank"><img src="/images/sponsors/logo_nyulmc_blue.jpg" alt="NYU Langone Medical Center" width="97" height="51" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nih.gov/" target="_blank"><img src="/images/sponsors/nih-logo.gif" alt="NIH" width="63" height="63" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://cancer.gov/" target="_blank"><img src="/images/sponsors/nci-logo.jpg" alt="NCI" width="89" height="56" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nigms.nih.gov/" target="_blank"><img src="/images/sponsors/nigms-logo.jpg" alt="NIGMS" width="63" height="75" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nidcr.nih.gov/" target="_blank"><img src="/images/sponsors/nidcr-logo.jpg" alt="NIDRC" width="58" height="72" vspace="2" border="0"></a>
													</td>
												</tr>
											</table>										
										</td>
									</tr>
								</table>
							</td>
							<td width="5"  height="5" bgcolor="#AAAAAA" ><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td width="620" style="padding: 25" valign="top" class="content">
<b>Proteomics Informatics Spring 2013 (BMSC-GA 4437)</b>
<p>
<b>Instructor:</b> David Feny&ouml;, Associate Professor<br>
<b>Contact information:</b> David@FenyoLab.org<br>



	<br>
	<br>
    <strong>Week 1</strong> Overview of proteomics (1/29/2013 at 4 pm in TRB 718)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-1.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-1.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<i>Reading list</i>
	<li>M.A. Gillette, S.A. Carr, <a href="pdf/13_gillette_nmeth.pdf" target="_blank"><strong>&quot;Quantitative analysis of peptides and proteins in biomedicine by targeted mass spectrometry&quot;</strong></a>, Nature Methods 10 (2013) 28–34.</li>
	<li>A. Bensimon, A.J.R. Heck R. Aebersold, <a href="pdf/12_bensimon_annurev-biochem.pdf" target="_blank"><strong>&quot;Mass Spectrometry–Based Proteomics and Network Biology&quot;</strong></a>, Annual Review of Biochemistry 81 (2012) 379-405.</li>


	
	

	<br>
	<br>
	<strong>Week 2</strong> Overview of mass spectrometry (2/5/2013 at 4 pm in TRB 718)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-2.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-2.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<a href="http://fenyolab.ionomix.com" target="_blank"><strong>Example data</strong></a> (<a href="http://youtube.googleapis.com/v/78N0AFjhmjc&hd=1" target="_blank">Slice video</a>)
	<br>
	<br>	
	<i>Reading list</i>
	<li>Beavis, R.C. & Chait, B.T. <a href="pdf/96_beavis_methods-enzymol.pdf" target="_blank"><strong>&quot;Matrix-assisted laser desorption ionization mass-spectrometry of proteins&quot;</strong></a> Meth. Enzymol 270, 519-551 (1996).</li>

	<li>Banks, J.F. & Whitehouse, C.M. <a href="pdf/96_banks_methods-enzymol.pdf" target="_blank"><strong>&quot;Electrospray ionization mass spectrometry&quot;</strong></a> Meth. Enzymol 270, 486-519 (1996).</li>

	<li>Chalkley, R. <a href="pdf/10_chalkley_meth-mol-biol.pdf" target="_blank"><strong>&quot;Instrumentation for LC-MS/MS in proteomics&quot;</strong></a> Methods Mol. Biol 658, 47-60 (2010).</li>


	<br>
	<br>
	<strong>Week 3</strong> Analysis of mass spectra: signal processing, peak finding, and isotope clusters (2/12/2013 at 4 pm in TRB 119)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-3.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-3.pptx" target="_blank">pptx</a>
	<br>	
	<br>	
	<i>Reading list</i>
	<li>Zhang, J., Gonzalez, E., Hestilow, T., Haskins, W. & Huang, Y. <a href="pdf/09_zhang_current-genomics.pdf" target="_blank"><strong>&quot;Review of peak detection algorithms in liquid-chromatography-mass spectrometry&quot;</strong></a> Curr. Genomics 10, 388-401 (2009).</li>

	<li>Yang, C., He, Z. & Yu, W. <a href="pdf/09-yang-BMC-bioinformatics.pdf" target="_blank"><strong>&quot;Comparison of public peak detection algorithms for MALDI mass spectrometry data analysis&quot;</strong></a> BMC Bioinformatics 10, 4 (2009).</li>

	
	<br>
	<br>
	<strong>Week 4</strong> Protein identification I: searching protein sequence collections and significance testing (2/19/2013 at 4 pm in TRB 718)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-4.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-4.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<i>Reading list</i>
	<li>Eriksson, J., Chait, B.T. & Feny&ouml;, D. <a href="/publications/pdf/00_eriksson_anal-chem.pdf" target="_blank"><strong>&quot;A statistical basis for testing the significance of mass spectrometric protein identification results&quot;</strong></a> Anal. Chem 72, 999-1005 (2000).</li>

	<li>Feny&ouml;, D. & Beavis, R.C. <a href="/publications/pdf/03_fenyo_anal-chem.pdf" target="_blank"><strong>&quot;A method for assessing the statistical significance of mass spectrometry-based protein identifications using general scoring schemes&quot;</strong></a> Anal. Chem 75, 768-774 (2003).</li>

	<li>Elias, J.E. & Gygi, S.P. <a href="pdf/07_elias_nature-meth.pdf" target="_blank"><strong>&quot;Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry&quot;</strong></a> Nat. Methods 4, 207-214 (2007).</li>

	
	<br>
	<br>
	<strong>Week 5</strong> Protein identification II: search engines and protein sequence databases (3/5/2013 at 4 pm in TRB 718)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-5.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-5.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<strong>Example data:</strong> 
	E. coli (<a href="data/ecoli_1.mgf" target="_blank">1</a>, <a href="data/ecoli_2.mgf" target="_blank">2</a>, <a href="data/ecoli_3.mgf" target="_blank">3</a>), 
	Yeast (<a href="data/yeast_1.mgf" target="_blank">1</a>, <a href="data/yeast_2.mgf" target="_blank">2</a>, <a href="data/yeast_3.mgf" target="_blank">3</a>), 
	Human (<a href="data/human_1.mgf" target="_blank">1</a>, <a href="data/human_2.mgf" target="_blank">2</a>, <a href="data/human_3.mgf" target="_blank">3</a>)
	<br>
	<br>
	<i>Reading list</i>
	<li>Eng, J.K., McCormack, A.L. & Yates III, J.R. <a href="pdf/94_eng_jasms.pdf" target="_blank"><strong>&quot;An approach to correlate tandem mass spectral data of peptides with amino acid sequences in a protein database</strong></a>, Journal of the American Society for Mass Spectrometry 5, 976-989 (1994).</li>

	<li>Craig, R. & Beavis, R.C. <a href="pdf/04_craig_bioinformatics.pdf" target="_blank"><strong>&quot;TANDEM: matching proteins with tandem mass spectra</strong></a>, Bioinformatics 20, 1466-1467 (2004).</li>

	<li>Craig, R., Cortens, J.C., Fenyo, D. & Beavis, R.C. <a href="/publications/pdf/06_craig_jpr.pdf" target="_blank"><strong>&quot;Using annotated peptide mass spectrum libraries for protein identification</strong></a>, J. Proteome Res 5, 1843-1849 (2006).</li>

	
	<br>
	<br>
	<strong>Week 6</strong> Protein identification III: de novo sequencing (3/12/2013 at 4 pm in TRB 718)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-6.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-6.pptx" target="_blank">pptx</a>

	<br>Instructor: Beatrix Ueberheide (Beatrix.Ueberheide@nyumc.org)
	<br>
	<br>
	<i>Reading list</i>
	<li>Seidler J, Zinn N, Boehm ME, Lehmann WD, <a href="pdf/10_seidler_proteomics.pdf" target="_blank"><strong>&quot;De novo sequencing of peptides by MS/MS</strong></a>, Proteomics 10 (2010) 634-49.</li>

	<li>Standing KG, <a href="pdf/03_standing_curr-opin-struct-biol.pdf" target="_blank"><strong>&quot;Peptide and protein de novo sequencing by mass spectrometry</strong></a>, Curr Opin Struct Biol. 13 (2003) 595-601.</li>




	<br>
	<br>
	<strong>Week 7</strong> Databases, data repositories and standardization (3/19/2013 at 4 pm in TRB 718)
	- <strong>Slides</strong>: <a href="slides/Proteomics-Informatics-Week-7.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-7.pptx" target="_blank">pptx</a>
	<br>
	<br>	
	<i>Reading list</i>
	<li>Craig R, Cortens JP, Beavis RC, <a href="/publications/pdf/04_craig_jpr.pdf" target="_blank"><strong>&quot;Open source system for analyzing, validating, and storing protein identification data&quot;</strong></a>, J Proteome Res. 3 (2004) 1234-42.</li>

	<li>D. Feny&ouml;, J. Eriksson, R. Beavis, <a href="/publications/pdf/10_fenyo_meth-mol-biol.pdf" target="_blank"><strong>&quot;Mass spectrometric protein identification using the global proteome machine&quot;</strong></a>, Methods Mol Biol 673 (2010) 189-202.</li>

	<li>CF Taylor, NW Paton, KS Lilley, P-A Binz, RK Julian Jr, AR Jones, W Zhu, R Apweiler, R Aebersold, EW Deutsch, MJ Dunn, AJR Heck, A Leitner, M Macht, M Mann, L Martens, TA Neubert, SD Patterson, P Ping, SL Seymour, P Souda, A Tsugita, J Vandekerckhove, TM Vondriska, JP Whitelegge, MR Wilkins, I Xenarios, JR Yates III, H Hermjakob, <a href="pdf/07_taylor_nbt.pdf" target="_blank"><strong>&quot;The minimum information about a proteomics experiment (MIAPE)&quot;</strong></a>, Nat Biotechnol. 25 (2007) 887-93.</li>

<font color="darkred">
	
	<br>
	<br>
	<strong>Week 8</strong> Protein quantitation I: Overview (4/2/2013 at 4 pm in TRB 718)
	<br>
</font>	
	<br>	
	<i>Reading list</i>
	<li>Domon, B. & Aebersold, R. <a href="pdf/10_domon_nbt.pdf" target="_blank"><strong>&quot;Options and considerations when selecting a quantitative proteomics strategy&quot;</strong></a>, Nat. Biotechnol 28, 710-721 (2010).</li>

	<li>Zhang, G. et al. <a href="/publications/pdf/10_zhang_methods-mol-biol.pdf" target="_blank"><strong>&quot;Protein quantitation using mass spectrometry&quot;</strong></a> Methods Mol. Biol 673, 211-222 (2010).</li>

	<li>Cox, J. & Mann, M. <a href="pdf/08_cox_nbt.pdf" target="_blank"><strong>&quot;MaxQuant enables high peptide identification rates, individualized p.p.b.-range mass accuracies and proteome-wide protein quantification&quot;</strong></a>, Nat Biotech 26, 1367-1372 (2008).</li>
	

<font color="#888888">	<br>
	<br>
	<strong>Week 9</strong> Protein quantitation II: Multiple Reaction Monitoring (4/9/2013 at 4 pm in TRB 718)
	<br>Instructor: Kelly Ruggles (Kelly.Ruggles@nyumc.org)
	<br>
	<br>	
	<i>Reading list</i>

	<li>MacLean, B. et al. Skyline: an open source document editor for creating and analyzing targeted proteomics experiments. Bioinformatics 26, 966-968 (2010).</li>

	
	<br>
	<br>
	<strong>Week 10</strong> Protein characterization: post-translational modifications, protein-protein interactions, cross-linking, top-down, non-covalent complexes (4/23/2013 at 4 pm in TRB 718)
	<br>
	<br>	
	<i>Reading list</i>
	<li>Trost, M., Bridon, G., Desjardins, M. & Thibault, P. Subcellular phosphoproteomics. Mass Spectrom. Rev. 29, 962-990 (2010).</li>
	<li>Petrotchenko, E.V. & Borchers, C.H. Crosslinking combined with mass spectrometry for structural proteomics. Mass Spectrom. Rev. 29, 862-876 (2010).</li>
	<li>A Leitner, R Reischl, T Walzthoeni, F Herzog, S Bohn, F F&ouml;rster, and R Aebersold, Expanding the Chemical Cross-Linking Toolbox by the Use of Multiple Proteases and Enrichment by Size Exclusion Chromatography, Mol Cell Proteomics 11 (2012)</li>


	<br>
	<br>
	<strong>Week 11</strong> Molecular Signatures (4/30/2013 at 4 pm in TRB 718)
	<br>
	<br>	
	<i>Reading list</i>
	<li>EF Petricoin III, AM Ardekani, BA Hitt, PJ Levine, VA Fusaro, SM Steinberg, GB Mills, C Simone, DA Fishman, EC Kohn, LA Liotta, Use of proteomic patterns in serum to identify ovarian cancer, Lancet 359 (2002) 572–77</li>
	<li>DF Ransohoff, Bias as a threat to the validity of cancer molecular-marker research, Nat Rev Cancer 5 (2005) 142-9.</li>
	<li>TA Addona, X Shi, H Keshishian, DR Mani, M Burgess, MA Gillette, KR Clauser, D Shen, GD Lewis, LA Farrell, MA Fifer, MS Sabatine, RE Gerszten & SA Carr, A pipeline that integrates the discovery and verification of plasma protein biomarkers reveals candidate markers for cardiovascular disease, 29 (2011) 635</li>


	<br>
	<br>
	<strong>Week 12</strong> Presentations of projects (5/7/2013 at 4 pm in TRB 718)
	<br>
	<br>	
	
	<br>
	<br>
	<strong>Week 13</strong> Presentations of projects (5/14/2013 at 4 pm in TRB 718)
	<br>
	<br>


</font>
								</p>
								<p>&nbsp;</p>
								<center>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
							</td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td bgcolor="#EEEEEE"><img src="/images/spacer.gif" border="0"></td>
						</tr>
					</table>
				</td>
			</tr>
		</table>
	</body>

</html>
