<html>
	<head>
		<title>Laboratory of Computational Proteomics - Proteomics Informatics Spring 2014</title>
		<link rel="stylesheet" type="text/css" href="/css/Style.css" />
	</head>

	<body bgcolor="#ffffff" bottommargin="0" leftmargin="0" marginheight="0" marginwidth="0" rightmargin="0" topmargin="0">
		<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
			<tr>
				<td>
					<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
						<tr>
							<td width="2"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="2" height="5" border="0"></td>
							<td width="6"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="6" height="5" border="0"></td>
							<td width="172" height="50" bgcolor="#EEEEEE" align="left" valign="middle"><img src="/images/logo.png" width="172" height="43" border="0"></td>
							<td width="1"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="4" height="5" border="0"></td>
							<td width="620" class="siteName" align="center">Laboratory of Computational Proteomics<br><font size=-1>Proteomics Informatics Spring 2014</font></td>
							<td width="5"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
						</tr>
					</table>
				</td>
			</tr>
			<tr>		
				<td valign="top">
					<table width="100%" height="100%" border="0" cellpadding="0" cellspacing="0">
						<tr>
							<td width="10"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="10" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="160" height="5" bgcolor="#AAAAAA" align="center" valign="top"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td width="620" height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="376" height="5" border="0"></td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
							<td  height="5" bgcolor="#EEEEEE" valign="top"><img src="/images/spacer.gif" width="5" height="5" border="0"></td>
						</tr>
						<tr>
							<td width="10" bgcolor="#EEEEEE"><img src="/images/spacer.gif" border="0"></td>
							<td width="5"  height="5" bgcolor="#EEEEEE"><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td width="160" valign="top" bgcolor="#EEEEEE" align="center">
								<table width="150" border="0" cellpadding="0" cellspacing="0" bgcolor="#EEEEEE">
									<tr>
										<td style="padding: 20 0 20 8;" align="center">
											<table width="100%" border="0" cellpadding="0" cellspacing="8">
												<tr>
													
													<td><a class="menu" href="/index.html">Overview</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/research/research.html">Research</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/members/members.html">Members</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/publications/publications.html">Publications</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/presentations/presentations.html">Presentations</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/tools/tools.html">Tools</a></td>
												</tr>
												<tr>
													<td><a class="menu" href="/contact/contact.html">Contact</a></td>
												</tr>
												<tr>
													<td>&nbsp;</td>
												</tr>
											</table>
										</td>
									</tr>
									<tr>
										<td style="padding: 0 0 0 0;" align="left">
											<table width="120" border="0" cellpadding="0" cellspacing="0">	
												<tr>
													<td><center>
														<a class="chibi" href="http://www.nyuinformatics.org/" target="_blank">Center for Health Informatics and Bioinformatics</font></a>
													</td>
												</tr>
												<tr>
													<td><center>
														&nbsp;
													</td>
												</tr>
												<tr>
													<td><center>
														<a class="chibi" href="http://www.nyuinformatics.org/services/hpcf" target="_blank">High Performance Computing Facility</font></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.med.nyu.edu/" target="_blank"><img src="/images/sponsors/logo_nyulmc_blue.jpg" alt="NYU Langone Medical Center" width="97" height="51" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nih.gov/" target="_blank"><img src="/images/sponsors/nih-logo.gif" alt="NIH" width="63" height="63" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://cancer.gov/" target="_blank"><img src="/images/sponsors/nci-logo.jpg" alt="NCI" width="89" height="56" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nigms.nih.gov/" target="_blank"><img src="/images/sponsors/nigms-logo.jpg" alt="NIGMS" width="63" height="75" vspace="2" border="0"></a>
													</td>
												</tr>
												<tr>
													<td><center>
														<a href="http://www.nist.gov/" target="_blank"><img src="/images/sponsors/nist-logo.jpg" alt="NIST" width="89" height="53" vspace="2" border="0"></a>
													</td>
												</tr>
											</table>										
										</td>
									</tr>
								</table>
							</td>
							<td width="5"  height="5" bgcolor="#AAAAAA" ><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td width="620" style="padding: 25" valign="top" class="content">
<b>Proteomics Informatics Spring 2014 (BMSC-GA 4437)</b>
<p>
<b>Course Director:</b> David Feny&ouml;, Associate Professor<br>
<b>Contact information:</b> David@FenyoLab.org<br>
<br>
<b>Instructors:</b><br>
David Feny&ouml; (David@FenyoLab.org)<br>
Himanshu Grover (Himanshu.Grover@nyumc.org) - Python Lab and Databases, Week 8 data repositories and standardization<br>
Kelly Ruggles (Kelly.Ruggles@nyumc.org) - Week 7 Protein quantitation II: Multiple Reaction Monitoring and Week 9 Proteogenomics<br>
Beatrix Ueberheide (Beatrix.Ueberheide@nyumc.org) - Week 5 de novo sequencing
<br><br><br>
This course will give an introduction of proteomics and mass spectrometry workflows, experimental design, and data analysis with a focus on algorithms for extracting information from experimental data. The following subjects will be covered in: (1) Protein identification (peptide mass fingerprinting, tandem mass spectrometry, database searching, spectrum library searching, de novo sequencing, significance testing); (2) Protein char­acterization (protein coverage, top-down proteomics, post-translational modifications, protein processing and degradation, protein complexes); (3) Protein quantitation (metabolic labeling - SILAC, chemical labeling, label-free quantitation, spectrum counting, stoichiometry, biomarker discovery and verification). Examples will be provided throughout the course on how the different approaches can be applied to investigate biological systems. The class will be structured to include hands-on practical techniques for analyzing relevant proteomics datasets. 


	<br>
	<br>
    <strong>Week 1</strong> Overview of proteomics (1/28/2014 at 4 pm in TRB 718)
	- <strong>Lecture Slides</strong>: <a href="slides/Proteomics-Informatics-Week-1.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-1.pptx" target="_blank">pptx</a>, <b>Python Slides:</b> <a href="slides/Proteomics-Informatics-Python-Week-1.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Python-Week-1.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<i>Reading list</i>
	<li>M.A. Gillette, S.A. Carr, <a href="pdf/13_gillette_nmeth.pdf" target="_blank"><strong>&quot;Quantitative analysis of peptides and proteins in biomedicine by targeted mass spectrometry&quot;</strong></a>, Nature Methods 10 (2013) 28–34.</li>
	<li>A. Bensimon, A.J.R. Heck R. Aebersold, <a href="pdf/12_bensimon_annurev-biochem.pdf" target="_blank"><strong>&quot;Mass Spectrometry–Based Proteomics and Network Biology&quot;</strong></a>, Annual Review of Biochemistry 81 (2012) 379-405.</li>


	
	

	<br>
	<br>
	<strong>Week 2</strong> Overview of mass spectrometry (2/4/2014 at 4 pm in TRB 718)
	- <strong>Lecture Slides</strong>: <a href="slides/Proteomics-Informatics-Week-2.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-2.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<a href="http://openslice.fenyolab.org/" target="_blank"><strong>Example data</strong></a> (<a href="http://youtube.googleapis.com/v/78N0AFjhmjc&hd=1" target="_blank">Slice video</a>)
	<br>
	<br>	
	<i>Reading list</i>
	<li>Beavis, R.C. & Chait, B.T. <a href="pdf/96_beavis_methods-enzymol.pdf" target="_blank"><strong>&quot;Matrix-assisted laser desorption ionization mass-spectrometry of proteins&quot;</strong></a> Meth. Enzymol 270, 519-551 (1996).</li>

	<li>Banks, J.F. & Whitehouse, C.M. <a href="pdf/96_banks_methods-enzymol.pdf" target="_blank"><strong>&quot;Electrospray ionization mass spectrometry&quot;</strong></a> Meth. Enzymol 270, 486-519 (1996).</li>

	<li>Chalkley, R. <a href="pdf/10_chalkley_meth-mol-biol.pdf" target="_blank"><strong>&quot;Instrumentation for LC-MS/MS in proteomics&quot;</strong></a> Methods Mol. Biol 658, 47-60 (2010).</li>


	<br>
	<br>
	<strong>Week 3</strong> Analysis of mass spectra: signal processing, peak finding, and isotope clusters (2/11/2014 at 4 pm in TRB 717)
	- <strong>Lecture Slides</strong>: <a href="slides/Proteomics-Informatics-Week-3.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-3.pptx" target="_blank">pptx</a>, <b>Python Slides:</b> <a href="slides/Proteomics-Informatics-Python-Week-3.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Python-Week-3.pptx" target="_blank">pptx</a>
	<br>	
	<br>	
	<i>Reading list</i>
	<li>Zhang, J., Gonzalez, E., Hestilow, T., Haskins, W. & Huang, Y. <a href="pdf/09_zhang_current-genomics.pdf" target="_blank"><strong>&quot;Review of peak detection algorithms in liquid-chromatography-mass spectrometry&quot;</strong></a> Curr. Genomics 10, 388-401 (2009).</li>

	<li>Yang, C., He, Z. & Yu, W. <a href="pdf/09-yang-BMC-bioinformatics.pdf" target="_blank"><strong>&quot;Comparison of public peak detection algorithms for MALDI mass spectrometry data analysis&quot;</strong></a> BMC Bioinformatics 10, 4 (2009).</li>

	
	<br>
	<br>
	<strong>Week 4</strong> Protein identification I: searching protein sequence collections and significance testing (2/18/2014 at 4 pm in TRB 718)
	- <strong>Lecture Slides</strong>: <a href="slides/Proteomics-Informatics-Week-4.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-4.pptx" target="_blank">pptx</a>, <b>Python Slides:</b> <a href="slides/Proteomics-Informatics-Python-Week-4.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Python-Week-4.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<strong>Example code:</strong> <a href="scripts/Proteomics-Informatics-Week-4-Ex1-py" target="_blank">Ex1.py</a>, <a href="scripts/Proteomics-Informatics-Week-4-computeMz-py" target="_blank">computeMz.py</a><br><br>
	<strong>Example data:</strong> 	
	E. coli (<a href="data/ecoli_1.mgf" target="_blank">1</a>, <a href="data/ecoli_2.mgf" target="_blank">2</a>, <a href="data/ecoli_3.mgf" target="_blank">3</a>), 
	Yeast (<a href="data/yeast_1.mgf" target="_blank">1</a>, <a href="data/yeast_2.mgf" target="_blank">2</a>, <a href="data/yeast_3.mgf" target="_blank">3</a>), 
	Human (<a href="data/human_1.mgf" target="_blank">1</a>, <a href="data/human_2.mgf" target="_blank">2</a>, <a href="data/human_3.mgf" target="_blank">3</a>)
	<br>
	<br>
	<i>Reading list</i>
	<li>Eriksson, J., Chait, B.T. & Feny&ouml;, D. <a href="/publications/pdf/00_eriksson_anal-chem.pdf" target="_blank"><strong>&quot;A statistical basis for testing the significance of mass spectrometric protein identification results&quot;</strong></a> Anal. Chem 72, 999-1005 (2000).</li>

	<li>Feny&ouml;, D. & Beavis, R.C. <a href="/publications/pdf/03_fenyo_anal-chem.pdf" target="_blank"><strong>&quot;A method for assessing the statistical significance of mass spectrometry-based protein identifications using general scoring schemes&quot;</strong></a> Anal. Chem 75, 768-774 (2003).</li>

	<li>Elias, J.E. & Gygi, S.P. <a href="pdf/07_elias_nature-meth.pdf" target="_blank"><strong>&quot;Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry&quot;</strong></a> Nat. Methods 4, 207-214 (2007).</li>

	
	<br>
	<br>
	<strong>Week 5</strong> Protein identification II: de novo sequencing (2/25/2014 at 4 pm in TRB 718)
	- <strong>Lecture Slides</strong>: <a href="slides/Proteomics-Informatics-Week-5.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-5.pptx" target="_blank">pptx</a>, <b>Python Slides:</b> <a href="slides/Proteomics-Informatics-Python-Week-5.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Python-Week-5.pptx" target="_blank">pptx</a>
	<br>Instructor: Beatrix Ueberheide (Beatrix.Ueberheide@nyumc.org)
	<br>
	<br>
	<strong>Example code:</strong> <a href="scripts/Proteomics-Informatics-Week-5-peptideFragmentation-py" target="_blank">peptideFragmentation.py</a><br><br>
	<strong>Example data:</strong> 	
	<a href="data/Proteomics-Informatics-Week-5-cheat sheet.pdf" target="_blank">Cheat sheet</a>, 
	<a href="data/Proteomics-Informatics-Week-5-Ex1-CAD-Spectrum.pdf" target="_blank">Example 1: CAD Spectrum</a>, 
	<a href="data/Proteomics-Informatics-Week-5-Ex1-HCD-spectrum.pdf" target="_blank">Example 1: HCD Spectrum</a>, 
	<a href="data/Proteomics-Informatics-Week-5-Ex2-CAD-spectra.pdf" target="_blank">Example 2: CAD Spectrum</a>, 
	<a href="data/Proteomics-Informatics-Week-5-Ex3-ETD-spectrum-1.pdf" target="_blank">Example 3: ETD Spectrum</a>, 
	<a href="data/Proteomics-Informatics-Week-5-Ex4-ETD-spectrum-2.pdf" target="_blank">Example 4: ETD Spectrum</a>.
	<br>
	<br>
	<i>Reading list</i>
	<li>Seidler J, Zinn N, Boehm ME, Lehmann WD, <a href="pdf/10_seidler_proteomics.pdf" target="_blank"><strong>&quot;De novo sequencing of peptides by MS/MS</strong></a>, Proteomics 10 (2010) 634-49.</li>
	<li>Standing KG, <a href="pdf/03_standing_curr-opin-struct-biol.pdf" target="_blank"><strong>&quot;Peptide and protein de novo sequencing by mass spectrometry</strong></a>, Curr Opin Struct Biol. 13 (2003) 595-601.</li>

<br>
	<br>
	<strong>Week 6</strong> Protein quantitation I: Overview (3/4/2014 at 4 pm in TRB 718)
	- <strong>Lecture Slides</strong>: <a href="slides/Proteomics-Informatics-Week-6.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Week-6.pptx" target="_blank">pptx</a>, <b>Python Slides:</b> <a href="slides/Proteomics-Informatics-Python-Week-6.pdf" target="_blank">pdf</a>, <a href="slides/Proteomics-Informatics-Python-Week-6.pptx" target="_blank">pptx</a>
	<br>
	<br>
	<strong>Example code:</strong> <a href="scripts/Proteomics-Informatics-Week-6-peptideFragmentation_withFunctions-py" target="_blank">peptideFragmentation_withFunctions.py</a>
	<br>
	<br>	
	<i>Reading list</i>
	<li>Domon, B. & Aebersold, R. <a href="pdf/10_domon_nbt.pdf" target="_blank"><strong>&quot;Options and considerations when selecting a quantitative proteomics strategy&quot;</strong></a>, Nat. Biotechnol 28, 710-721 (2010).</li>
	<li>Zhang, G. et al. <a href="/publications/pdf/10_zhang_methods-mol-biol.pdf" target="_blank"><strong>&quot;Protein quantitation using mass spectrometry&quot;</strong></a> Methods Mol. Biol 673, 211-222 (2010).</li>
	
	
	<br>
	<br>
	<strong>Week 7</strong> Protein quantitation II: Multiple Reaction Monitoring (3/11/2014 at 4 pm in TRB 718)
	<a href="https://nyumc.webex.com/nyumc/j.php?ED=288536572&UID=0&RT=MiMxMQ%3D%3D" taget="_blank"><b>Webex</b></a>
	<br>Instructor: Kelly Ruggles (Kelly.Ruggles@nyumc.org)
	<br>
	<br>	
	<i>Reading list</i>

	<li>D.R. Mani, S.E. Abbatiello, S.A. Carr, <a href="pdf/12_mani_bmc-bioinf.pdf" target="_blank"><strong>&quot;Statistical characterization of multiple-reaction monitoring mass spectrometry (MRM-MS) assays for quantitative proteomics&quot;</strong></a>, BMC Bioinformatics 13 (2012) S9.</li>
	<li>A. Maiolica, M.A. Jünger, I. Ezkurdi, R. Aebersold, <a href="pdf/12_maiolica_j-prot.pdf" target="_blank"><strong>&quot;Targeted proteome investigation via selected reaction monitoring mass spectrometry&quot;</strong></a>, Journal of Proteomics 75 (2012) 3495-3513.</li>
	<li>MacLean, B. et al. <a href="pdf/10_maclean_bioinformatics.pdf" target="_blank"><strong>&quot;Skyline: an open source document editor for creating and analyzing targeted proteomics experiments&quot;</strong></a>, Bioinformatics 26, 966-968 (2010).</li>


	<br>
	<br>
	<strong>Week 8</strong> Databases, data repositories and standardization (3/18/2014 at 4 pm in TRB 718)
	<br>Instructor: Himanshu Grover (Himanshu.Grover@nyumc.org)
	<br>
	<br>	
	<i>Reading list</i>
	<li>Craig R, Cortens JP, Beavis RC, <a href="/publications/pdf/04_craig_jpr.pdf" target="_blank"><strong>&quot;Open source system for analyzing, validating, and storing protein identification data&quot;</strong></a>, J Proteome Res. 3 (2004) 1234-42.</li>
	<li>D. Feny&ouml;, J. Eriksson, R. Beavis, <a href="/publications/pdf/10_fenyo_methods-mol-biol.pdf" target="_blank"><strong>&quot;Mass spectrometric protein identification using the global proteome machine&quot;</strong></a>, Methods Mol Biol 673 (2010) 189-202.</li>
	<li>CF Taylor, NW Paton, KS Lilley, P-A Binz, RK Julian Jr, AR Jones, W Zhu, R Apweiler, R Aebersold, EW Deutsch, MJ Dunn, AJR Heck, A Leitner, M Macht, M Mann, L Martens, TA Neubert, SD Patterson, P Ping, SL Seymour, P Souda, A Tsugita, J Vandekerckhove, TM Vondriska, JP Whitelegge, MR Wilkins, I Xenarios, JR Yates III, H Hermjakob, <a href="pdf/07_taylor_nbt.pdf" target="_blank"><strong>&quot;The minimum information about a proteomics experiment (MIAPE)&quot;</strong></a>, Nat Biotechnol. 25 (2007) 887-93.</li>


	<br>
	<br>
	<strong>Week 9</strong> Proteogenomics (3/25/2014 at 4 pm in TRB 718)
	<br>Instructor: Kelly Ruggles (Kelly.Ruggles@nyumc.org)
	<br>
	<br>
	<i>Reading list</i>

	
	<br>
	<br>
	<strong>Week 10</strong> Protein characterization I: post-translational modifications (4/1/2014 at 4 pm in TRB 718)
	<br>
	<br>	
	<i>Reading list</i>
	<li>Trost, M., Bridon, G., Desjardins, M. & Thibault, P. <a href="pdf/10_trost_mass-spec-rev.pdf" target="_blank"><strong>&quot;Subcellular phosphoproteomics&quot;</strong></a>, Mass Spectrom. Rev. 29, 962-990 (2010).</li>

	<br>
	<br>
	<strong>Week 11</strong> Protein characterization II: Protein interactions (4/10/2014 at 4 pm in TRB 718)
	<br>
	<br>	
	<i>Reading list</i>
	<li>A Leitner, R Reischl, T Walzthoeni, F Herzog, S Bohn, F F&ouml;rster, and R Aebersold, <a href="pdf/12_leitner_mcp.pdf" target="_blank"><strong>&quot;Expanding the Chemical Cross-Linking Toolbox by the Use of Multiple Proteases and Enrichment by Size Exclusion Chromatography&quot;</strong></a>, Mol Cell Proteomics 11 (2012)</li>

	<br>
	<br>
	<strong>Week 12</strong> Molecular Signatures (4/17/2014 at 4 pm in TRB 718)
	<br>
	<br>	
	<i>Reading list</i>
	<li>EF Petricoin III, AM Ardekani, BA Hitt, PJ Levine, VA Fusaro, SM Steinberg, GB Mills, C Simone, DA Fishman, EC Kohn, LA Liotta, <a href="pdf/02_petricoin_lancet.pdf" target="_blank"><strong>&quot;Use of proteomic patterns in serum to identify ovarian cancer&quot;</strong></a>, Lancet 359 (2002) 572–77</li>
	<li>DF Ransohoff, <a href="pdf/05_ransohoff_nrc.pdf" target="_blank"><strong>&quot;Bias as a threat to the validity of cancer molecular-marker research&quot;</strong></a>, Nat Rev Cancer 5 (2005) 142-9.</li>
	<li>TA Addona, X Shi, H Keshishian, DR Mani, M Burgess, MA Gillette, KR Clauser, D Shen, GD Lewis, LA Farrell, MA Fifer, MS Sabatine, RE Gerszten & SA Carr, <a href="pdf/11_addona_nbt.pdf" target="_blank"><strong>&quot;A pipeline that integrates the discovery and verification of plasma protein biomarkers reveals candidate markers for cardiovascular disease&quot;</strong></a>, Nature Biotechnology 29 (2011) 635</li>


	<br>
	<br>
	<strong>Week 13</strong> Presentations of projects (4/22/2014 at 4 pm in TRB 718)
	<br>
	<br>

	
<font color="darkred">
</font>	
<font color="#888888">	
</font>

<br>
<br>



								</p>
								<p>&nbsp;</p>
								<center>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
								<p>&nbsp;</p>
							</td>
							<td width="5"  height="5" bgcolor="#AAAAAA"><img src="/images/spacer.gif" width="5" height="35" border="0"></td>
							<td bgcolor="#EEEEEE"><img src="/images/spacer.gif" border="0"></td>
						</tr>
					</table>
				</td>
			</tr>
		</table>
	</body>

</html>
